bottles/backend/state.py

```diff
@@ -97,7 +97,7 @@ def subtitle(self, value: str):
     def stream_update(self, received_size: int = 0, total_size: int = 0, status: Status = None):
         """This is a default subtitle updating handler for streaming downloading progress"""
         match status:
-            case Status.DONE, Status.FAILED:
+            case Status.DONE | Status.FAILED:
```

Comment:
ref: https://peps.python.org/pep-0636/#or-patterns

Since I was not aware of this one and maybe other are interested.

Reply:
it's intended to be a OR condition when I wrote it before
but comma will make it to match sequence
so fix it here.
```
Python 3.10.10 (main, Mar  5 2023, 22:26:53) [GCC 12.2.1 20230201] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> match 1:
...  case 1,2:
...   print('yes')
...  case _:
...   print('no')
... 
no
>>> match 1:
...  case 1|2:
...   print('yes')
...  case _:
...   print('no')
... 
yes
>>>
```